components/NavigationDrawer/src/private/MDCBottomDrawerContainerViewController.m

```diff
@@ -593,23 +613,27 @@ - (void)cacheLayoutCalculations {
 }
 
 - (void)cacheLayoutCalculationsWithAddedContentHeight:(CGFloat)addedContentHeight {
-  CGFloat contentHeight = self.contentViewController.preferredContentSize.height;
-  _contentVCPreferredContentSizeHeightCached = contentHeight;
   CGFloat contentHeaderHeight = self.contentHeaderHeight;
   CGFloat containerHeight = self.presentingViewBounds.size.height;
+  CGFloat contentHeight = self.contentViewController.preferredContentSize.height;
+  if ([self isAccessibilityMode]) {
+    contentHeight = MAX(contentHeight, containerHeight - self.topHeaderHeight);
```

Comment:
Why is this only necessary in accessibility mode?

Reply:
because in a11y mode, the drawer has to *always* be presented in full screen, even if there is less than a full screen of content. Therefore, in a11y mode, the content height is either the container height (screen height), or if the content height is bigger than the entire screen, then just leave it as is.